frontend-implicaction/src/app/job/components/jobs-list/jobs-list.component.html

```diff
@@ -41,7 +41,8 @@
               (onChange)="onSortChange($event)"
               [(ngModel)]="selectedOrderCode"
               [options]="orderByEnums"
-              name="selectedOrder"
+              styleClass="no-outline"
+              name="selectedOrder "
```

Comment:
```suggestion
              name="selectedOrder"
```

Un petit espace qui traîne